src/main/resources/db/migration/V5__new_table_updates.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE `updates` (
+  `uuid` CHAR(36) NOT NULL,
+  `namespace` CHAR(200) NOT NULL,
+  `name` VARCHAR(200) NOT NULL,
+  `description` VARCHAR(600) NOT NULL,
+  `created_at` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `updated_at` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+
```

Comment:
We need an optional external ID here

Reply:
Alright, I added an `externalID` with a unique constraint on `(ns, externalId)`.